pkg/kubelet/dockershim/docker_container.go

```diff
@@ -347,7 +347,7 @@ func (ds *dockerService) ContainerStatus(_ context.Context, req *runtimeapi.Cont
 	// Convert the image id to a pullable id.
 	ir, err := ds.client.InspectImageByID(r.Image)
 	if err != nil {
-		return nil, fmt.Errorf("unable to inspect docker image %q while inspecting docker container %q: %v", r.Image, containerID, err)
+		klog.Errorf("unable to inspect docker image %q while inspecting docker container %q: %v", r.Image, containerID, err)
```

Comment:
I'm a little nervous about ignoring this error. I see how ignoring it could be useful when the image has already been cleaned up, but are there other times when we would want failures to raise an error?

Reply:
why  ignore it nor return err？